functions/go/inflate-helm-chart/third_party/sigs.k8s.io/kustomize/api/types/helmchartargs.go

```diff
@@ -73,50 +71,3 @@ type HelmChart struct {
 	// Defaults to 'false'.
 	IncludeCRDs bool `json:"includeCRDs,omitempty" yaml:"includeCRDs,omitempty"`
 }
-
-// HelmChartArgs contains arguments to helm.
```

Comment:
I'd suggest leave these unchanged. Otherwise, it may be a pain when you try to rebase our fork to newer `kustomize/api`

Reply:
thanks for the suggestion, I readded it